manifest.yaml

```diff
@@ -0,0 +1,6 @@
+iso:
+  uefi:
+    - oci:registry.opensuse.org/isv/rancher/elemental/teal53/15.4/rancher/elemental-live-grub2-efi-image/5.3
+  image:
+    - oci:registry.opensuse.org/isv/rancher/elemental/teal53/15.4/rancher/elemental-live-grub2/5.3
```

Comment:
cant we bundle all of the artifacts into just one package?

Reply:
probably, but note we still need to build both in OBS as on is for the EFI image and the efi image contents are also required into the ISO filesystem.

This is just a draft, I am far from happy with this approach, but at least the OBS containers don't make my eyes bleed with strange and weird scripts.